CHANGELOG.md

```diff
@@ -48,6 +48,7 @@
 - The `[name=...]`, `[time=...]` and `[color=...]` tags may now be used anywhere in the document and not just inside of blockquotes and lists.
 - The <i class="fa fa-picture-o"/> (add image) and <i class="fa fa-link"/> (add link) toolbar buttons, put selected links directly in the `()` instead of the `[]` part of the generated markdown
 - The help dialog has multiple tabs, and is a bit more organized.
+- Use KaTeX instead of MathJax.
```

Comment:
someone posted a comparison link to the chat. Maybe this would be great information for someone who reads this change.

Reply:
or perhaps this one (if it is not already somewhere else, maybe the features page would also be a good place for this)
https://katex.org/docs/supported.html

Reply:
the features page is currently not in this repo

Reply:
I added a link to our FAQ page.